example/go.sum

```diff
@@ -1,502 +1,67 @@
-cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
this file was probably deleted / not regenerated by accident?

Reply:
Nope. I just ran `go mod tidy`.

Reply:
Maybe not in the `example/` dir? Otherwise there should still be a go.sum 🤔